src/app.rs

```diff
@@ -22,11 +22,15 @@ use crate::shell::window::WindowHandle;
 use crate::shell::{init, runloop, Error as PlatformError, WindowBuilder};
 use crate::win_handler::AppState;
 use crate::window::{Window, WindowId};
-use crate::{theme, Data, DruidHandler, LocalizedString, MenuDesc, Widget};
+use crate::{theme, Data, DruidHandler, Env, LocalizedString, MenuDesc, Widget};
+
+/// A function that modifies the initial environment.
+type EnvSetupFn = dyn Fn(&mut Env);
```

Comment:
Would FnOnce be a more precise bound?

Reply:
Yea I think that's reasonable, I could imagine needing this to be reusable at some point in the future but we can change it back if that happens. :)